app/views/subjects/_form.html.haml

```diff
@@ -9,8 +9,7 @@
     = f.submit
   - if @subject.errors.any?
     #error_explaination
-    %h2= pluralize(@subject.errors.count, 'error')
-    prohibited this subject from being saved:
+    %h2 #{pluralize @subject.errors.count, 'error'} prohibited this subject from being saved:
```

Comment:
Yeah - so i just modified to 
`#error_explaination`
(two spaces)`%h2 #{pluralize @subject.errors.count, 'error'} prohibited this subject from being saved:`